src/App.js

```diff
@@ -125,61 +125,55 @@ function Root() {
-          </Route>
-          <Route exact path="/decode-tx">
-            <NavigationRoute internalScreen={DecodeTx} />
-          </Route>
-          <Route exact path="/transactions">
-            <NavigationRoute internalScreen={TransactionList} />
-          </Route>
-          <Route exact path="/tokens">
-            <NavigationRoute internalScreen={TokenList} />
-          </Route>
-          <Route exact path="/token_balances">
-            <NavigationRoute internalScreen={TokenBalancesList} />
-          </Route>
-          <Route exact path="/token_balances">
-            <NavigationRoute internalScreen={TokenBalancesList} />
-          </Route>
-          <Route exact path="/blocks">
-            <NavigationRoute internalScreen={BlockList} />
-          </Route>
-          <Route exact path="/dag" component={Dag}>
-            <NavigationRoute internalScreen={Dag} />
-          </Route>
-          <Route exact path="/features">
-            <NavigationRoute internalScreen={FeatureList} />
-          </Route>
-          <Route exact path="/network/:peerId?">
-            <NavigationRoute internalScreen={PeerAdmin} />
-          </Route>
-          <Route exact path="/statistics">
-            <NavigationRoute internalScreen={Dashboard} />
-          </Route>
-          <Route exact path="/token_detail/:tokenUID">
-            <NavigationRoute internalScreen={TokenDetail} />
-          </Route>
-          <Route exact path="/address/:address">
-            <NavigationRoute internalScreen={AddressDetail} />
-          </Route>
-          <Route exact path="/nano_contract/detail/:nc_id" component={NanoContractDetail}>
-            <NavigationRoute internalScreen={NanoContractDetail} />
-          </Route>
-          <Route exact path="/blueprint/detail/:blueprint_id">
-            <NavigationRoute internalScreen={BlueprintDetail} />
-          </Route>
-          <Route exact path="">
-            <NavigationRoute internalScreen={DashboardTx} />
-          </Route>
-        </Switch>
-      </Router>
+      <BrowserRouter>
+        <Routes>
+          <Route
+            path="/transaction/:id"
+            element={<NavigationRoute internalScreen={TransactionDetail} />}
+          />
+          <Route path="/push-tx" element={<NavigationRoute internalScreen={PushTx} />} />
+          <Route path="/decode-tx" element={<NavigationRoute internalScreen={DecodeTx} />} />
+          <Route
+            path="/transactions"
+            element={<NavigationRoute internalScreen={TransactionList} />}
+          />
+          <Route path="/tokens" element={<NavigationRoute internalScreen={TokenList} />} />
+          <Route
+            path="/token_balances"
+            element={<NavigationRoute internalScreen={TokenBalancesList} />}
+          />
+          <Route
+            path="/token_balances"
+            element={<NavigationRoute internalScreen={TokenBalancesList} />}
+          />
```

Comment:
Is this duplicated?

Reply:
Yes it is. This was fixed in a later PR through https://github.com/HathorNetwork/hathor-explorer/pull/384/commits/a2a2dff8574df1070dde9a6e8acc27eb912ef5b2